src/pip-delete-this-directory.txt

```diff
@@ -0,0 +1,5 @@
+This file is placed here by pip to indicate the source was put
```

Comment:
These two files should not have been committed. Always check your:
* git status before you commit
* Your PR before you send it out for review